frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigCard.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+
+const useStyles = makeStyles((theme) => ({
+  padding: {
+    marginTop: 50,
+  },
+  card: {
+    height: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    paddingTop: theme.spacing(2),
+    paddingBottom: theme.spacing(2),
+    marginTop: 50
+  },
+  cardContent: {
+    flexGrow: 1,
+  },
+}));
+
+export default function ConfigurCard(card) {
```

Comment:
can we change this to `ConfigCard` instead of `ConfigurCard` to make the class name the same as the file naem?